provider/ghasedak/ghasedak.go

```diff
@@ -0,0 +1,60 @@
+package ghasedak
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"time"
+	"github.com/messagebird/sachet"
+)
+
+// Retrieving required data from 'ghasedak' sections of config.yaml
+type Config struct {
+	APIToken     string   `yaml:"api_token"`
+	PhoneNumbers []string `yaml:"phone_numbers"`
+}
+
+// Creating the KaveNegar to contain provider data
+type Ghasedak struct {
+	Config
+	HTTPClient *http.Client // The HTTP client to send requests on
+}
+
+// Ghasedak creates and returns a new Ghasedak struct
+func NewGhasedak(config Config) *Ghasedak {
+	return &Ghasedak{
+		config,
+		&http.Client{Timeout: time.Second * 20},
+	}
+}
+
+// Building the API and call the Ghasedak endpoint to send SMS to the configured receptor from config.yaml
+func (ns *Ghasedak) Send(message sachet.Message) error {
+	url := "https://api.ghasedak.me/v2/sms/send/pair"
+	payload := strings.NewReader("message=" + message.Text + "&receptor=" + strings.Join(message.To, ","))

```

Comment:
There may be encoding problems here. 
You should use https://pkg.go.dev/net/url#Values.Encode

Reply:
I Use Encode in payload request